json_annotation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.7.1-dev+1
+
+- Require Dart SDK `>=2.18.0`.
```

Comment:
4.7.1 hasn't been shipped yet (hence the -dev) – so you can put this in the blew section